### DISCUSSION TITLE
Send brightness values out via API

### DISCUSSION THREAD

**Discussion by serivas** [xanderfrangos/twinkle-tray#915] [Ideas]
I love the schedule functionality. It would be incredibly helpful if we could somehow send this as an event, webhook, or via API to other (local) apps. This way we could use Twinkle Tray schedules to set off other automations.

**Comment by xanderfrangos** [comment]
There's currently UDP and named pipe support for sending commands to Twinkle Tray, but not for Twinkle Tray sending out updates. I don't know that I'd want to implement an HTTP server, but would sending updates via the UDP server or named pipe work? If so, what sort of events would be helpful?
